pkg/restapi/vc/operation/operations.go

```diff
@@ -27,16 +28,20 @@ import (
 )
 
 const (
-	credential               = "credential"
+	credentialStore   = "credential"
+	credentialContext = "https://www.w3.org/2018/credentials/v1"
+
+	// endpoints
 	createCredentialEndpoint = "/credential"
+	verifyCredentialEndpoint = "/verify"
 	createProfileEndpoint    = "/profile"
-	credentialContext        = "https://www.w3.org/2018/credentials/v1"
+	getProfileEndpoint       = "/profile/{" + profilePathVariable
```

Comment:
whats the { for?

Reply:
Good question @talwinder50, I just re-arranged constants so endpoints are together.